### DISCUSSION TITLE
Can't open some files via default application

### DISCUSSION THREAD

**Discussion by MijikHna** [sxyazi/yazi#1630] [1. Q&A]
I have installed `yazi` via `nix package manager`. 

I have `Okular` installed via OS package manager additionally I have `PDF Master Editor` via `flatpak` installed.

Problem: If I want to open pdf file, something strange happens. PDF application (in my case PDF Master Editor) starts but closes immediately. (I can see the process in task manager). Sometimes ,it seems to me, If I start another application (doesn't matter which one), PDF Master Editor shows up with the file I have opened in yazi.

The same applies to .png files and Gwenview, but gwenview just closes immediately without a remaining process in task manager. 

.odt files work. Yazi starts Libre Office correctly.

Did someone faced similar problem?

**Comment by nguyenhoanghipe** [comment]
It would be great if someone could help look at this issue.
On my Mac, I installed IINA as a music player, but I couldn't open any video/music files with it.

**Reply by sevillaarvin** [reply]
> It would be great if someone could help look at this issue. On my Mac, I installed IINA as a music player, but I couldn't open any video/music files with it.

This worked for me on macos:

`~/.config/yazi.toml`
```toml
[[opener.play]]
for = "macos"
block = true
# orphan = true
run = 'iina "$@"'
```
